src/views/Users/ListUsers/index.jsx

```diff
@@ -58,15 +57,16 @@ function ListUsers() {
               scheduledPermissions={users[user].scheduledPermissions}
             />
           ))}
-          <Tooltip title="Add User">
-            <Fab
-              color="primary"
-              className={classes.fab}
-              classes={{ root: classes.fab }}
-              onClick={handleUserAdd}>
-              <PlusIcon />
-            </Fab>
-          </Tooltip>
+          <Link to="/users/create">
+            <Tooltip title="Add User">
+              <Fab
+                color="primary"
+                className={classes.fab}
+                classes={{ root: classes.fab }}>
+                <PlusIcon />
+              </Fab>
+            </Tooltip>
+          </Link>
```

Comment:
Navigating to http://localhost:9000/users/create throws an error `User does not exist!`.

Reply:
D'oh, silly mistake.